components/brave_sync/features.cc

```diff
@@ -12,5 +12,9 @@ namespace features {
 
 BASE_FEATURE(kBraveSync, "BraveSync", base::FEATURE_ENABLED_BY_DEFAULT);
 
+BASE_FEATURE(kBraveSyncHistoryDiagnostics,
```

Comment:
a comment would be nice :)

Reply:
thanks, updated at https://github.com/brave/brave-core/pull/16478/commits/78d6b7498161a2125aa72960a6aef2a490a87c02